modules/troubleshooting-network-observability-loki-resource-exhausted.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-troubleshooting-loki-resource-exhausted_{context}"]
+= Troubleshooting Loki ResourceExhausted error
+Loki may return a "ResourceExhausted" error when network flow data sent by Network Observability exceeds the configured maximum message size. When using the Loki operator, this maximum message size is configured to 100 MiB. 
+
+.Procedure
+. Navigate to *Operators* -> *Installed Operators*, viewing *All projects* from the *Project* dropdown.
+. Look under the *Provided APIs* heading for the Netwok Observability Operator, select *Flow Collector* the the *YAML view* tab. 
+.. If you are using the Loki Operator, check that the `spec.loki.batchSize` value does not exceed, or come close to 100 MiB. You must keep a margin, such as 98 MiB max.
+.. If you are not using the Loki Operator, and are instead using Grafana Loki, check that the `grpc_server_max_recv_msg_size` [link:https://grafana.com/docs/loki/latest/configure/#server[Grafana Loki server setting]] is higher than the `FlowCollector` `spec.loki.batchSize` configuration. If it is not, you must either increase the server-side limit, or decrease the FlowCollector batch size setting to make it lower than the limit. 
```

Comment:
@jotak is the distinction here between Red Hat Loki Operator and Grafana Loki Operator correct? In the [Jira](https://issues.redhat.com/browse/NETOBSERV-1148), I wasn't sure if you meant "when you aren't using Loki" to mean you aren't using Loki at all, or if you meant you're using a different kind of Loki. I assumed the later because you linked to the Grafana reference.

Reply:
Yes I meant they're using a different installation method for Loki. I don't know if saying "and are instead using Grafana Loki" is correct, bc even with redhat loki operator they would be using Grafana Loki in the end of the day, it's only the installation method that differs. That's probably a bit nitpicking :-)

Reply:
A couple of things then, one is since this is technically an unsupported installation method, do we need to say _anything_ about it here? 

Second, if we do need to mention it, I revised to this:

> If you are using a Loki Operator installation method that is different from the Red Hat Loki Operator, check that the `grpc_server_max_recv_msg_size` link:https://grafana.com/docs/loki/latest/configure/#server[Grafana Loki server setting] is higher than the `FlowCollector` `spec.loki.batchSize` configuration.

Reply:
it's not unsupported, just not advertized... so I would still keep that sentence

Also:

> If you are using a Loki ~~Operator~~ installation method...

because in this case, Loki would be installed without an operator.

Reply:
(other than that, lgtm!)

Reply:
I guess unsupported is a strong word. What I meant is that it isn't Red Hat supported. It works, as far as functionality goes but if a customer calls with this issue and they've chosen another Loki installation method, then they would be on their own. I guess we can help them a little here though. Won't hurt :) I added back the "such as Grafana Loki" and removed the Operator per your point that Loki doesn't have an Operator in this case. 

Thank you!